SeaPublicWebsite.Data/Migrations/20220614163252_AddDedicatedPrimaryKeyToEpcTable.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace SeaPublicWebsite.Data.Migrations
+{
+    public partial class AddDedicatedPrimaryKeyToEpcTable : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                    name: "Id",
+                    table: "Epc",
+                    type: "integer",
+                    nullable: false,
+                    defaultValue: 0)
+                .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn);

```

Comment:
I know this is the way to go, but I have to ask: Why is it we can't update an epc with the same ID and instead have to create a new one? As in, why did the database complain?